src/components/TextField/__tests__/__snapshots__/textField-snapshot-tests.jest.js.snap

```diff
@@ -60,22 +60,22 @@ exports[`TextField renders correctly when disabled 1`] = `
 exports[`TextField renders correctly when loading 1`] = `
 <div
   aria-busy={true}
-  className="input-component"
+  className="textField input-component"
   role=""
 >
   <div
-    className="input-component__label--wrapper"
+    className="labelWrapper input-component__label--wrapper"
   >
     <div
-      className="input-component__input-wrapper input-component__input-wrapper--size-small"
+      className="inputWrapper input-component__input-wrapper input-component__input-wrapper--size-small"
```

Comment:
missing sizeSmall css-modules style here, cause no changes in TextFieldConstants - snapshots are a good way to spot these kind of errors